Tests/js/views/actions/assets/ez-translateactionview-tests.js

```diff
@@ -0,0 +1,273 @@
+                method: 'toJSON',
+                returns: {}
+            });
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            this.view = new Y.eZ.TranslateActionView({
+                container: '.container',
+                actionId: 'translate',
+                label: 'Translations',
+                disabled: false,
+                content: this.contentMock,
+                location: this.locationMock
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should hide the view when clicking outside of the view": function () {
+            this.view.render();
+            this.view.set('expanded', true);
+            Y.one('.click-somewhere').simulate('click');
+
+            Assert.isFalse(this.view.get('expanded'), "The view should be hidden");
+        },
+
+        "Should leave the view when clicking outside of the view": function () {
+            this.view.render();
+            this.view.set('expanded', true);
+            this.view.set('expanded', false);
+            Y.one('.click-somewhere').simulate('click');
+
+            Assert.isFalse(this.view.get('expanded'), "The view should be hidden");
+        }
+    });
+
+    hintTest = new Y.Test.Case({
+        name: 'eZ Translate Action View hint test',
+
+        setUp: function () {
+            this.contentMock = new Mock();
+            this.locationMock = new Mock();
+            this.versionMock = new Mock();
+
+            Mock.expect(this.contentMock, {
+                method: 'get',
+                args: ['currentVersion'],
+                returns: this.versionMock
+            });
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            this.view = new Y.eZ.TranslateActionView({
+                container: '.container',
+                actionId: 'translate',
+                label: 'Translations',
+                disabled: false,
+                content: this.contentMock,
+                location: this.locationMock
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should display all translations in button hint": function () {
+            var translationsList = ['Diego Armando Maradona Franco', 'Dennis Nicolaas Maria Bergkamp'],
+                hintExpected = 'Diego Armando Maradona Franco, Dennis Nicolaas Maria Bergkamp';
+
+            Mock.expect(this.versionMock, {
+                method: 'getTranslationsList',
+                returns: translationsList
+            });
+
+            this.view = new Y.eZ.TranslateActionView({
```

Comment:
the view is instantiated in the `setUp`, you should not recreate a new instance here.